### DISCUSSION TITLE
同一个PagImageView设置不同pag文件首次会空白一会问题

### DISCUSSION THREAD

**Discussion by gentle821** [Tencent/libpag#2355] [Q&A] [answered]
同一个pagImageView每点击一次设置不同的pag文件 通过setpathAsync回调后进行执行pagImageView.play后有一段空白期  我看pagfile回调了都
请问下这种情况官方是如何处理

**Comment by domchen** [comment, marked as answer]
网络加载是有延迟的。你要立即切换，应该自己预加载好离线的PAG文件，设置本地路径进去，并且立马调用flush方法。
